src/libs/OptionsListUtils.js

```diff
@@ -549,6 +550,7 @@ function getOptions(
         forcePolicyNamePreview = false,
         includeOwnedWorkspaceChats = false,
         includeThreads = false,
+        includeTasks = false,
```

Comment:
Hey, this PR caused the bug - [Task name is not appearing in search page](https://github.com/Expensify/App/issues/21565#issuecomment-1607555779)